<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundPressureLevel</class>
 <widget class="QWidget" name="SoundPressureLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layout_main">
     <item>
      <layout class="QHBoxLayout" name="layout_decibel">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Decibel:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_decibel">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>TBD</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Weighting:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_weighting">
         <item>
          <property name="text">
           <string>dB(A)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dB(B)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dB(C)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dB(D)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>dB(Z)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ITU-R 468</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>FFT Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_fft_size">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>128 (80 SPS)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256 (40 SPS)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512 (20 SPS)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024 (10 SPS)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_graph"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
